src/commands/commandHandlers/SlashCommandRegister.ts

```diff
@@ -0,0 +1,47 @@
+import SlashCommand from './SlashCommand.js';
+import SlashCommandRegistry from './SlashCommandRegistry.js';
+import { REST } from '@discordjs/rest';
+import { Routes } from 'discord-api-types/v9';
+import { Client, Collection, ChatInputCommandInteraction } from 'discord.js';
+
+export default class SlashCommandRegister {
+	public static async registerCommands( client: Client, token: string ) {
+		client.guilds.cache.forEach( async fetchedGuild => {
+			await fetchedGuild.fetch();
+
+			// eslint-disable-next-line @typescript-eslint/no-explicit-any
+			const commands: any[] = [];
```

Comment:
I dislike the fact that this is untyped, is there no better option?

Reply:
Yes, it can be typed. It just took a bit of digging to find what to type it to.